apps/site/src/styles/noir.css

```
@@ -0,0 +1,492 @@
+  color: var(--noir-accent);
+  transition: color var(--transition-fast);
+}
+
+html:root a:hover {
+  color: var(--noir-text-primary);
+}
+
+/* Kill gradient text utility under Noir */
+html:root .text-gradient {
+  background: none;
+  -webkit-background-clip: unset;
```

Stylelint (17.14.0)

[error] Vendor-prefixed property "-webkit-background-clip" (property-no-vendor-prefix)

(property-no-vendor-prefix)